tests/tests.yaml

```diff
@@ -35,9 +35,10 @@ answer_tests:
   local_gizmo_002:
     - yt/frontends/gizmo/tests/test_outputs.py
 
-  local_halos_001:
+  local_halos_002:
```

Comment:
The cached answers deprecated since we have changed the Hubble constant for the AHF data. Is it the right thing to do to bump the answer number here in order to generate new answers?